From 57d95ad3579dbfcae23f8a07f9a0539872e98cfe Mon Sep 17 00:00:00 2001
From: Pradeep Sriram <pradeep.sriram@amlogic.com>
Date: Tue, 10 Nov 2020 16:20:01 -0800
Subject: [PATCH 10/10] RDK: splash logo path fix

PD#SWPL-36538

Problem: splash logo not working

Solution: use vendor partition

Verify: Build pass

Change-Id: I414a0c285e02105f0d5c1b2d2595e54b616bf7a7
Signed-off-by: Pradeep Sriram <pradeep.sriram@amlogic.com>
---
 board/amlogic/configs/tm2_t962x3_ab301_v1.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/board/amlogic/configs/tm2_t962x3_ab301_v1.h b/board/amlogic/configs/tm2_t962x3_ab301_v1.h
index 190309e039..743d203662 100644
--- a/board/amlogic/configs/tm2_t962x3_ab301_v1.h
+++ b/board/amlogic/configs/tm2_t962x3_ab301_v1.h
@@ -105,7 +105,7 @@
         "fb_addr=0x3d800000\0" \
         "fb_width=1920\0" \
         "fb_height=1080\0" \
-        "board_logo_part=odm\0" \
+        "board_logo_part=vendor\0" \
         "frac_rate_policy=1\0" \
         "usb_burning=update 1000\0" \
         "otg_device=1\0"\
-- 
2.29.0

